src/screens/resetPassword/components/resetEmail/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { validateEmail } from '../../../../utils/validateFields';
+import {
+  CardContainer,
+  ContentContainer,
+  TypographyContainer,
+  TypographyHeader,
+  TypographySub,
+  TypographySubContainer,
+  InputContainer,
+  EmailTextFieldContainer,
+  EmailTextField,
+  EmailError,
+  ButtonContainer,
+  ConfirmButton,
+  ReturnButton,
+} from './styles';
+
+type Props = {
+  email: string;
+  emailError: string;
+  setEmailError(email: string): void;
+  handleEmailChange(e: React.ChangeEvent<HTMLInputElement>): void;
+  handleConfirmEmailClick(): void;
+  handleLoginClick(): void;
+};
+
+const ResetEmail = ({
+  email,
+  emailError,
+  setEmailError,
+  handleEmailChange,
+  handleConfirmEmailClick,
+  handleLoginClick,
+}: Props) => {
+  const EmailErrorHelper = () => {
+    if (emailError) {
+      return <EmailError error>{emailError}</EmailError>;
+    }
+    return <></>;
+  };
+  return (
+    <CardContainer>
+      <ContentContainer>
+        <TypographyContainer>
+          <TypographySubContainer>
+            <TypographyHeader>Recuperação de Senha</TypographyHeader>
+            <TypographySub>
+              Digite abaixo seu e-mail para iniciarmos o processo de recuperação
+              de senha.
+            </TypographySub>
+          </TypographySubContainer>
+        </TypographyContainer>
+        <InputContainer>
+          <EmailTextFieldContainer>
+            <EmailTextField
```

Comment:
Colocar o type também ajuda no autocomplete do browser:

```suggestion
            <EmailTextField
              type="email"
```